resources/public/api/conf/5.0/schemas/foreign_property_cumulative_summary_retrieve/def1/response.json

```diff
@@ -0,0 +1,198 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "For TY 2025-26 and after",
+  "description": "This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.",
+  "type": "object",
+  "properties": {
+    "submittedOn": {
+      "type": "string",
+      "description": "The date the foreign property were submitted, in the format YYYY-MM-DDThh:mm:ss.SSSZ",
+      "example": "2025-07-07T10:59:47.544Z"
+    },
+    "fromDate": {
+      "type": "string",
+      "description": "The first day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD.",
+      "example": "2025-04-06"
+    },
+    "toDate": {
+      "type": "string",
+      "description": "The last day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD.",
+      "example": "2025-07-05"
+    },
+    "foreignProperty": {
+      "type": "array",
+      "description": "Array holding the income and expenses of the user's foreign property business for the period.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "countryCode": {
+            "type": "string",
+            "description": "A three-letter code that represents a country name. This must be an ISO 3166-1 Alpha-3 value.",
+            "example": "FRA"
+          },
+          "income": {
+            "type": "object",
+            "description": "Object holding the income details for the period.",
+            "properties": {
+              "rentIncome": {
+                "type": "object",
+                "description": "Object holding the rent income.",
+                "properties": {
+                  "rentAmount": {
+                    "type": "number",
+                    "description": "Total rent and other income from property. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                    "multipleOf": 0.01,
+                    "minimum": 0.00,
+                    "maximum": 99999999999.99,
+                    "example": 5000.99
+                  }
+                },
+                "additionalProperties": false
+              },
+              "foreignTaxCreditRelief": {
```

Comment:
The field `foreignTaxCreditRelief` is marked as mandatory or required in our current specification for retrieve. However, it is listed as optional in the create and amend endpoint. I’m concerned that if this field is not supplied in the create and amend endpoint, it could break the retrieve endpoint. I will clarify if this field in the retrieve endpoint should be updated to optional. The downstream specifications for both endpoints have also marked this field as optional, which raises additional concerns.

If this is indeed meant to be optional in our specification, then it will not only need to be updated here, but the code will also require changes.

Reply:
As confirmed, this field should be optional. Please update accordingly, thanks. I've also noticed several changes are needed for field descriptions, documentation scenarios, and other aspects of the create and amend endpoint. I'll address those in a separate PR to avoid making this one unnecessarily large. For now, can you please just address the specific points I've highlighted? Thanks.